webui/react/src/services/apiConfig.ts

```diff
@@ -115,10 +115,10 @@ export const getUsers: HttpApi<EmptyParams, DetailedUser[]> = {
 
 /* Info */
 
-export const getInfo: HttpApi<EmptyParams, DeterminedInfo> = {
-  httpOptions: () => ({ url: '/info' }),
+export const getInfo: DetApi<EmptyParams, Api.V1GetMasterResponse, DeterminedInfo> = {
   name: 'getInfo',
-  postProcess: (response) => jsonToDeterminedInfo(response.data),
+  postProcess: (response) => jsonToDeterminedInfo(response),
+  request: () => detApi.Cluster.determinedGetMaster(),
```

Comment:
**blocking:** I think this needs to be the internal API? http://latest-master.determined.ai:8080/docs/rest-api/#/Internal/Determined_GetTelemetry